src/javascript/app/ng-wig/ng-wig-plugin-adapter.component.spec.js

```diff
@@ -0,0 +1,74 @@
+describe('component: ngWigPlugin', () => {
+    let component;
+    let scope;
+    let plugin;
+    let execCommand;
+    let editMode;
+    let disabled;
+    let options;
+    let content;
+    let element;
+    let compile;
+
+    beforeEach(module('ngWig'));
+
+    beforeEach(inject((_$componentController_, _$rootScope_, _$compile_) => {
+        scope = _$rootScope_.$new();
+        compile = _$compile_;
+
+        plugin = {
+            name: 'myPlugin',
+            pluginName: 'my-plugin'
+        };
+        editMode = false;
+        disabled = false;
+        options = ['Option 1', 'Option 2'];
+        content = 'Fake text';
```

Comment:
we can extend it later if we need to overwrite variables. 

beforeEach is good when you need really redefine the variable. 
That's why I suggest to keep this block clean: to see from the first look what should be set(or reset) for each test.